### ISSUE TITLE
Summary view should differentiate 0-hour weeks from unconfirmed weeks

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#781]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
Summary view shows [blank] weeks for both unconfirmed and confirmed 0-hour weeks.
0-hour weeks should show 0.00, to differentiate them from unconfirmed weeks.

### Description
Derivative of #776, #777

#### Example: my week 52, confirmed with 0 hours
![image](https://user-images.githubusercontent.com/7300548/103740968-83674900-4ff8-11eb-87e0-d43010e2ed2d.png)

#### Shows up as [blank] hours in the summary view
![image](https://user-images.githubusercontent.com/7300548/103741035-a691f880-4ff8-11eb-8cfa-52b513d2115f.png)

### Why fix it?
To aid team leads and billing responsibles in making sure everyone has confirmed their hours.

### Related
#841
